src/components/ConversationSettings/BotsSettings.vue

```diff
@@ -0,0 +1,164 @@
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div class="bots-settings">
+		<p class="bots-settings__hint">
+			{{ t('spreed', 'Bots are available for installation for administrators from Apps tab, and could be enabled by moderators for specific chats') }}
+		</p>
+
+		<ul v-if="bots?.length" class="bots-settings__list">
+			<li v-for="bot in bots"
+				:key="bot.id"
+				class="bots-settings__item">
+				<div class="bots-settings__item-info">
+					<span class="bots-settings__item-name">
+						{{ bot.name }}
+					</span>
+					<span class="bots-settings__item-description">
+						{{ bot.description ?? t('spreed', 'Description is not provided') }}
+					</span>
+				</div>
+				<NcButton class="bots-settings__item-button"
+					:type="bot.state ? 'primary' : 'secondary'"
+					:disabled="isBotForceEnabled(bot)"
+					@click="toggleBotState(bot)">
+					{{ toggleButtonTitle(bot) }}
+				</NcButton>
+			</li>
+		</ul>
+		<p v-else class="bots-settings__hint">
+			{{ t('spreed', 'There are currently no available bots for this conversation') }}
+		</p>
+	</div>
+</template>
+
+<script>
+
+import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
+
+import { BOT } from '../../constants.js'
+import { useBotsStore } from '../../stores/bots.js'
+
+export default {
+	name: 'BotsSettings',
+
+	components: {
+		NcButton,
+	},
+
+	props: {
+		/**
+		 * The conversation's token
+		 */
+		token: {
+			type: String,
+			required: true,
+		},
+	},
+
+	setup() {
+		const botsStore = useBotsStore()
+
+		return {
+			botsStore,
+		}
+	},
+
+	computed: {
+		bots() {
+			return this.botsStore.getBots(this.token)
+		},
+	},
+
+	created() {
+		this.botsStore.updateBotsInformation(this.token)
+	},
+
+	methods: {
+		isBotForceEnabled(bot) {
+			return bot.state === BOT.STATE.FORCE_ENABLED
+		},
+
+		toggleBotState(bot) {
+			if (this.isBotForceEnabled(bot)) {
+				return
+			}
+
+			this.botsStore.toggleBotState(this.token, bot)
+		},
+
+		toggleButtonTitle(bot) {
+			if (this.isBotForceEnabled(bot)) {
+				return t('spreed', 'Enabled')
+			}
+
+			return bot.state === BOT.STATE.ENABLED ? t('spreed', 'Disable') : t('spreed', 'Enable')
+		},
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+.bots-settings {
+	&__hint {
+		margin-bottom: calc(var(--default-grid-baseline) * 2);
+		color: var(--color-text-lighter);
+	}
+
+	&__list {
+	}
+
+	&__item {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		padding: 4px 0;
+
+		&:not(:last-child) {
+			border-bottom: 1px solid var(--color-background-darker);
+		}
+
+		&-info {
+			display: flex;
+			flex-direction: column;
+			max-width: 80%;
+		}
+
+		&-name {
+			font-size: var(--default-font-size);
+			font-weight: bold;
+			color: var(--color-main-text);
+		}
+
+		&-description {
+			font-size: var(--default-font-size);
+			color: var(--color-main-text);
```

Comment:
```suggestion
			color: var(--color-max-contrast);
```

(90% sure on the var name)